src/main/java/dinkplugin/DinkPlugin.java

```diff
@@ -143,6 +188,14 @@ public void onLootReceived(LootReceived lootReceived) {
     @Subscribe
     public void onVarbitChanged(VarbitChanged event) {
         diaryNotifier.onVarbitChanged(event);
+
+        if (event.getVarbitId() == CombatTaskNotifier.COMBAT_TASK_REPEAT_POPUP && event.getValue() > 0 && config.notifyCombatTask()) {
+            log.warn("Repeat popups for Combat Achievements is enabled; Dink Combat Task notifier will repeatedly fire on each completion.");
```

Comment:
Do we keep these as `log.warn`?
I imagine if they're changed to send a chat message it'd be spammy on world hop/login but worth considering, but maybe for a separate PR once this has been in a stable release

Reply:
Yeah, went for `warn` as a decent middleground between `debug` (won't appear in client.log without `--debug`) and chat (would be spammy if the setting was actually desired) - but can be revised later too

Reply:
Improved in https://github.com/pajlads/DinkPlugin/pull/92/commits/ae2aaba18dcb98e1e74efc6e4418d80f5bb707c4